recipe/patches/0007-Update-Abseil-to-the-latest-LTS-replace-or-with-in-d.patch

```diff
@@ -0,0 +1,450 @@
+From 96cd7d051fa6e0df83efcee259efec1092faf1c3 Mon Sep 17 00:00:00 2001
+From: David Dunleavy <ddunleavy@google.com>
+Date: Thu, 13 Apr 2023 10:02:13 -0700
+Subject: [PATCH 7/9] Update Abseil to the latest LTS, replace `or` with `||`
+ in data_format_ops.cc
+
+Current build requires a more recent Abseil with AnyInvocable, otherwise the build raise the following error:
```

Comment:
Could you leave a comment in `meta.yaml` where this commit comes from?

Reply:
I couldn't find a PR that matches it; thus I didn't add a link.

Reply:
Added a comment about the commit.